translation-web/src/main/java/eu/europeana/api/translation/web/service/TranslationWebService.java

```diff
@@ -38,7 +38,7 @@ public TranslationWebService(TranslationServiceProvider translationServiceProvid
 
   public TranslationResponse translate(TranslationRequest translationRequest)
       throws EuropeanaI18nApiException {
-    List<TranslationObj> translObjs = buildTranslationObjectList(translationRequest);
+    List<TranslationString> translObjs = buildTranslationObjectList(translationRequest);
```

Comment:
the field name should be updated after reamig the type. simply translations might be more appropriate